service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/IotHubConnectionStringBuilder.java

```diff
@@ -78,23 +103,19 @@ public static IotHubConnectionString createConnectionString(String hostName, Aut
      * @param iotHubConnectionString The target object for deserialization
      * @throws IOException This exception is thrown if the parsing failed
```

Comment:
it doesn't throw this exception anymore right?
on a separate note, if I extend this class, won't I be able to override this method? what I am getting at is that if we remove the `throws IOException` the overriden method will break. or does it?

Reply:
Thankfully static methods cannot be overriden, so we don't need to worry about that

Reply:
I'll remove the javadoc for the no longer thrown exception though